release-notes.md

```diff
@@ -8,12 +8,26 @@ Here are the release notes for the Experience Platform Mobile SDKs and Adobe Pla
 
 The following updates were made in this release:
 
+**iOS Core 2.3.5**
+
+* Added the support for attach data rules consequence.
+* Added the support for boolean type comparison for `Exist` or `Not Exist` rules condition.
+
+**Android Core 1.4.5**
+
+* Added the support for attach data rules consequence.
+* Added the support for boolean type comparison for `Equals` and `Not Equals` rules condition.
+* Fixed a bug where the `Exist` and `Not Exist` rules condition may not work for a `List` or `Map` type value.
+* Fixed a bug where it might cause a crash on some Android devices when fetching remote config from Launch.
```

Comment:
Please reword as follows:
"Fixed a bug that, when fetching the remote config from Launch, might cause a crash on some Android devices .